sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -167,6 +168,7 @@ class CAFMaker : public art::EDProducer {
   double fSubRunPOT;
   double fTotalSinglePOT;
   double fTotalEvents;
+  std::vector<caf::SRBNBInfo> bnb_info_store;
```

Comment:
Can we give this a name following the convention? `fBNBInfo` is probably fine. And a comment like `///< Store detailed BNB info to save into the first StandardRecord of the output file`

Reply:
Yep, my bad I forgot to follow the convention here